addons/hr/models/hr_employee_base.py

```diff
@@ -153,8 +155,10 @@ def _compute_presence_state(self):
             if check_login:
                 if 'online' in str(employee.user_id.im_status):
                     state = 'present'
-                elif 'offline' in str(employee.user_id.im_status) and employee.id not in working_now_list:
-                    state = 'absent'
+                elif 'offline' in str(employee.user_id.im_status):
+                    state = 'not_working' if employee.id not in working_now_list and employee.resource_calendar_id else 'absent'
```

Comment:
I think this has to be:
```
elif 'offline' in str(employee.user_id.im_status) and employee.id not in working_now_list:
    state = 'not_working' if employee.resource_calendar_id else 'absent'
```
to keep the same absent behaviour as before, no?

Reply:
Yes, it should be like that.

Thanks :slightly_smiling_face: